### DISCUSSION TITLE
如何生成whisper.cpp的不同平台可执行文件

### DISCUSSION THREAD

**Discussion by YT-Chowww** [ZuodaoTech/everyone-can-use-english#1163] [Q&A] [answered]
我看现在最新的项目已经不用whisper.cpp了，但还是想请教一下，是如何编译出不同平台(arm64,x86,windows,mac,linux)可执行文件的。
@an-lee 
Thanks!

**Comment by an-lee** [comment]
https://github.com/ggerganov/whisper.cpp 这里都有说明。

你是想在 Enjoy 里用 whipser.cpp 还是什么？新版 Enjoy 里的 whisper 用不了了吗？

**Reply by YT-Chowww** [reply]
我是把whisper.cpp项目的代码做了一下修改，但是对c++项目不太熟悉，想了解一下enjoy项目是如何把whisper.cpp编译成不平台(mac,Windows,linux)的可执行代码，就是lib/whisper目录下的文件。

比如是通过交叉编译工具，还是容器化的方案？

感谢回复！

**Reply by an-lee** [reply, marked as answer]
就是在不同平台编译一下，项目 README 上有编译说明吧。

**Reply by YT-Chowww** [reply]
了解~非常感谢！
